spec/kitchen/provisioner/chef_base_spec.rb

```diff
@@ -284,6 +293,25 @@ def calculate_path(path, _opts = {})
         cmd
       end
 
+      it "will use chef_omnibus_cache to provide a cache directory" do
```

Comment:
Any chance we could also include a test case where the user provides install_options other than -d? Maybe -v?